tsl/test/expected/compression_bgw.out

```diff
@@ -223,3 +223,158 @@ SELECT COUNT(*) AS dropped_chunks_count
 SELECT add_compression_policy AS job_id
   FROM add_compression_policy('conditions', INTERVAL '1 day') \gset
 CALL run_job(:job_id);
+\i include/recompress_basic.sql
```

Comment:
I will just note that most of the added testing here seems to be manual compression/recompression and not related to background jobs. Only found one call to `add_compression_policy`, so the question is if this is the appropriate test file to have the majority of these tests?